docs/ide/reference/how-to-use-the-keyboard-exclusively.md

```diff
@@ -31,13 +31,13 @@ For a full listing of command shortcut keys in Visual Studio, see [Default keybo
 ::: moniker-end
 
 > [!NOTE]
-> Depending on your settings or the edition of Visual Studio you use, the dialog boxes and menu commands you see might differ from those described in Help. To change your settings, choose **Import and Export Settings** on the **Tools** menu. For more information, see [Reset settings](../environment-settings.md#reset-settings).
+> Depending on your settings or the edition of Visual Studio you use, the dialog boxes and menu commands you see might differ from those described in Help. To change your settings, select **Import and Export Settings** in the **Tools** menu. For more information, see [Reset settings](../environment-settings.md#reset-settings).
```

Comment:
We appreciate your taking the time to create a PR! We'd like to ask for a few changes, though. Starting with the following:

Please note that our convention is that items are "on" a menu. See the examples in here: [https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#multiple-input-methods-and-branching-within-procedures](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-alternative-input-methods#multiple-input-methods-and-branching-within-procedures)